sympy/polys/domains/__init__.py

```diff
@@ -1,10 +1,10 @@
 """Implementation of mathematical domains. """
 
 __all__ = ['Domain', 'FiniteField', 'IntegerRing', 'RationalField',
-        'RealField', 'ComplexField', 'PythonFiniteField', 'GMPYFiniteField',
-        'PythonIntegerRing', 'GMPYIntegerRing', 'PythonRational',
-        'GMPYRationalField', 'AlgebraicField', 'PolynomialRing',
-        'FractionField', 'ExpressionDomain', 'PythonRational']
+           'RealField', 'ComplexField', 'PythonFiniteField', 'GMPYFiniteField',
```

Comment:
This is the only new line that has been made longer than 78. Others were long comments that are still long.

Reply:
Yes,  then, I regroup those lines.